pyface/ui/qt4/workbench/workbench_window_layout.py

```diff
@@ -292,7 +292,7 @@ def resolve_id(id):
         self._qt4_editor_area.restoreState(editor_layout, resolve_id)
 
     def get_toolkit_memento(self):
-        return (0, dict(geometry=str(self.window.control.saveGeometry())))
+        return (0, dict(geometry=self.window.control.saveGeometry()))
```

Comment:
It might be nice to take the opportunity to change this to the clearer:

    return (0, {'geometry': self.window.control.saveGeometry()})